<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1313</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AlexBuy's</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="viewSW">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(243, 243, 243);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLabel" name="label">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="pixmap">
            <pixmap resource="Logo.qrc">:/Sources/Logo.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="0">
             <widget class="QGroupBox" name="signUpGB">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>DejaVu Sans</family>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="title">
               <string>Sign In</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QLineEdit" name="newUserLE">
                 <property name="placeholderText">
                  <string>User</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="emailLE">
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="placeholderText">
                  <string>e-mail</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="newPassWordLE">
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                 <property name="placeholderText">
                  <string>Password</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="signUpPB">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>SIgn In</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QGroupBox" name="loginGB">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>DejaVu Sans</family>
                <pointsize>11</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="title">
               <string>LogIn</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QLineEdit" name="userNameLE">
                 <property name="placeholderText">
                  <string>User</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="passWordLE">
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                 <property name="placeholderText">
                  <string>Password</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="loginPB">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Login</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QScrollArea" name="scrollArea">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>90</y>
          <width>1291</width>
          <height>561</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Manjari</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1275</width>
           <height>545</height>
          </rect>
         </property>
         <property name="focusPolicy">
          <enum>Qt::WheelFocus</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QGridLayout" name="productLayout"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QLabel" name="logoLB">
        <property name="geometry">
         <rect>
          <x>1120</x>
          <y>0</y>
          <width>151</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="Logo.qrc">:/Sources/Logo.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QSplitter" name="splitter">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>1101</width>
          <height>51</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="departmentsLB">
            <property name="font">
             <font>
              <family>Manjari</family>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>DEPARTMENTS</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="departmentsCB">
            <property name="font">
             <font>
              <family>Manjari</family>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">selection-background-color: rgb(114, 159, 207);</string>
            </property>
            <property name="currentText">
             <string>All Departments</string>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <property name="maxVisibleItems">
             <number>6</number>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
            </property>
            <property name="minimumContentsLength">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string>All Departments</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Food and Drinks</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Books</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Electronics</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Home and Kitchen</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sport and outdoors</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="sortLB">
            <property name="font">
             <font>
              <family>Manjari</family>
             </font>
            </property>
            <property name="text">
             <string>SORT MODE</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="sortCB">
            <property name="styleSheet">
             <string notr="true">selection-background-color: rgb(114, 159, 207);</string>
            </property>
            <property name="editable">
             <bool>false</bool>
            </property>
            <property name="currentText">
             <string>None</string>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <property name="insertPolicy">
             <enum>QComboBox::InsertAfterCurrent</enum>
            </property>
            <property name="minimumContentsLength">
             <number>0</number>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Ascending</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Descending</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="searchLB">
            <property name="font">
             <font>
              <family>Manjari</family>
             </font>
            </property>
            <property name="text">
             <string>SEARCH PRODUCT</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="searchLE">
            <property name="placeholderText">
             <string>Product...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1313</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Logo.qrc"/>
 </resources>
 <connections/>
</ui>
